### ISSUE TITLE
Wallet CLI commands

### ISSUE THREAD

**Issue by austinabell** [ChainSafe/forest#537]
**Task summary**

dependent on #465 and just provides a CLI interface for the functionality exposed over rpc

commands options reference (from lotus):
     new          Generate a new key of the given type
     list         List wallet address
     balance      Get account balance
     export       export keys
     import       import keys
     default      Get default wallet address
     set-default  Set default wallet address

**Specification reference**

**Other information and links**

**Comment by amerameen** [comment]
On hold for now, since we don't have HTTP support, and we are focused on interop. Will come back to this after interop.

**Comment by connormullett** [comment]
Update: 7/11 commands working. Import, export, sign, verify are still WIP
